driver-core/src/test/functional/com/mongodb/operation/ChangeStreamOperationSpecification.groovy

```diff
@@ -331,6 +331,33 @@ class ChangeStreamOperationSpecification extends OperationFunctionalSpecificatio
         waitForLastRelease(getCluster())
     }
 
+    @IgnoreIf({ !serverVersionAtLeast([4, 0, 1]) })
```

Comment:
Looks like we have a test for each operation type, so let's add one for rename as well.

Reply:
the test for rename is in the last commit!

Reply:
acknowledged